### ISSUE TITLE
Actions: "Go" job failing due to TinyGo issue

### ISSUE THREAD

**Issue by case-fastly** [zonedb/zonedb#960]
As of the Go 1.22 release, our `go.yaml` jobs are failing here:

```
Run tinygo test -v -bench . ./

cannot resolve packages: requires go version 1.18 through 1.21, got go1.22

Error: Process completed with exit code 1.
```

This may be related: https://github.com/tinygo-org/tinygo/issues/4120

**Comment by ydnar** [comment]
Fixed by #961. Note to update the workflow file when TinyGo releases an update supporting Go 1.22.

**Comment by case-fastly** [comment]
Thanks!
